pages/reference-sdk-react-hooks/usesendtransaction.mdx

```diff
@@ -0,0 +1,137 @@
+import { Tabs } from 'nextra/components'
+
+# `useSendTransaction`
+
+`(params?: UseSendTransactionParams): UseSendTransactionReturnType`
+
```

Comment:
Missing a short description before jumping to the example.

Reply:
Done